### PR TITLE
Fixes from #37

### PR DESCRIPTION
**Pull request by imphil**
To reduce the scope of #37 a bit here's a PR with the fixes only without the version updates.

### CODE REVIEW

File: lowrisc-toolchain-gcc-rv32imcb.config [hunk 1]
```diff
@@ -25,9 +25,6 @@ CT_CC_GCC_STATIC_LIBSTDCXX=y
 # CT_CC_GCC_LDBL_128 is not set
 CT_CC_LANG_CXX=y
 CT_DEBUG_GDB=y
-CT_GDB_V_8_2=y
-# CT_GDB_CROSS_PYTHON is not set
-CT_ISL_V_0_20=y
```
Comment (luismarques) [inline lowrisc-toolchain-gcc-rv32imcb.config:30]: Isn't this version-change related?

**Review by luismarques — COMMENTED** [review]
Overall LGTM. Just double-check if the ct-ng config changes should really be included, as commented in-line.

File: lowrisc-toolchain-gcc-rv32imcb.config (hunk 1 above)
Comment (imphil) [reply lowrisc-toolchain-gcc-rv32imcb.config:30]: I wasn't sure and thought I'd give it a try :smile: Turns out CI fails now, so let's bring it back.

**Review by luismarques — APPROVED** [review]
LGTM.

**Pull request merged**